hermes/pkg/hermes/hermes.go

```diff
@@ -385,7 +391,7 @@ func (h *Hermes) Run(ctx context.Context, options ...Option) error {
 	if c.config != "" {
 		cmd = append(cmd, fmt.Sprintf("--%s=%s", FlagConfig, c.config))
 	}
-
+	cmd = append(cmd, "--json")
```

Comment:
I think that the `--json` flag should be optional because the Hermes App will be used in context where the output should be read by developers instead of log parsers. It would be a nice feature to optionally opt in for JSON output.

```suggestion
	if c.enableJSON {
		cmd = append(cmd, "--json")
	}
```

Reply:
But we need to parse the result of the requests most of the time. And we already hame clear logs, better than ts-relayer

Reply:
Does this mean that this app both captures the output of hermes and prints it out on stdout or just parses the output and prints out logs of its own if needed?

Reply:
No, we capture the output from Hermes in json, parse and get the result, and show our own logs. Only the start command, its prints the logs directly in the stdout

Reply:
The `ignite relayer hermes keys list` command also prints the list of keys. If `--json` is used then the output won't be user friendly.

If JSON support would be optional it could be enabled only for those commands that required it using an option like `hermes.EnableJSON()` when calling Hermes.

Reply:
Then could we just isolate `hermes` and always parse-reprint the results for every command instead of having exceptions or optional json flag?